### PR TITLE
proofread section 'introduction'

### PR DESCRIPTION
**Pull request by MarcusParadies**

### CODE REVIEW

File: introduction.tex [hunk 1]
```diff
@@ -79,37 +79,37 @@ \section{Relevance to Industry}
  \item \textbf{Graph database systems} (\eg Neo4j, InfiniteGraph, Sparksee,
      Titan) are novel technologies aimed at storing directed and labeled
      graphs. They support graph traverals, typically by means of APIs, though
-     some of them also support some sort of graph oriented query language (\eg
+     some of them also support dedicated graph-oriented query languages (\eg
      Neo4j's Cypher). These systems' internal structures are typically designed
-     to store dynamic graphs that change over time.  They ofter support
+     to store dynamic graphs that change over time.  They ofter support for
```
Comment (szarnyasg) [inline introduction.tex]: ofter -> offer

File: introduction.tex [hunk 2]
```diff
@@ -79,37 +79,37 @@ \section{Relevance to Industry}
...
      DBpedia). Their architecture is both single-machine and clustered, and
      they will likely target Interactive and Business Intelligence workloads.
- \item \textbf{Relational database systems} (\eg Postgres, MySQL, Oracle, DB2,
-     SQLserver, Virtuoso, MonetDB, Vectorwise, Vertica, but also Hive and
-     Impala) treat data as relational, and queries are formulated in SQL and/or
-     PL/SQL. Both single-machine and cluster systems exist.  They  do not
-     normally support recursion, or stateful recursive algorithms, which makes
-     them not at home in the Graph Analytics workloads
+ \item \textbf{Relational database systems} (\eg Postgres, MySQL, Oracle, IBM DB2,
+     SQLServer, Virtuoso, MonetDB, Vectorwise, Vertica, but also Hive and
```
Comment (szarnyasg) [inline introduction.tex]: I'd write "Microsoft SQL Server"

**Comment by ArnauPrat** [comment]
The text is definitely much better now. Once the comments from @szarnyasg  are applied, I personally approve the pull request

**Comment by ArnauPrat** [comment]
What's left for this to be merged?

**Pull request merged**